apps/google-analytics-4/src/locations/ConfigScreen.tsx

```diff
@@ -64,17 +66,52 @@ const styles = {
 };
 
 const ConfigScreen = () => {
-  const [parameters, setParameters] = useState<AppInstallationParameters>({});
+  const [parameters, setParameters] = useState<AppInstallationParameters>({
+    serviceAccountKey: null,
+    serviceAccountKeyId: null,
+  });
+
+  const [newServiceAccountKey, setNewServiceAccountKey] = useState<ServiceAccountKey | null>(null);
+  const [newServiceAccountKeyId, setNewServiceAccountKeyId] = useState<ServiceAccountKeyId | null>(
+    null
+  );
+  const [isInvalidServiceAccountKeyFile, setIsInvalidServiceAccountKeyFile] =
```

Comment:
Can we flip this to `isValid` to make it easier to read? Negative values need additional when reading expressions

Reply:
Yeah, I think it's better. I flipped it to match the input of Forma36 but i think it's much easier to reason about if we just flip it at the end when we feed it to the component.

Reply:
Actually for this, if you don't mind I'll tackle the change in a subsequent PR. Since i have a few PRs stacked and this actually touches logic in a few files it's just easier to isolate the change to avoid merge conflicts. (The cleanup task is captured in INTEG-99)